.travis.yml

```diff
@@ -63,8 +59,8 @@ services:
   - docker
 
 script:
-  - make ci-generate ci-build ci-test ci-deploy ci-docker
-  - GO111MODULE="on" make ci-soups
+  # - make ci-generate ci-build ci-test ci-deploy
```

Comment:
shall we remove this line completely now?

Reply:
yep, it can be removed